accel-pppd/ctrl/sstp/sstp.c

```diff
@@ -1956,6 +1994,7 @@ static int sstp_read(struct triton_md_handler_t *h)
 	return 0;
 
 drop:
+	n = sstp_write(h);
```

Comment:
I'm afraid it's ok place for error case, but not ok for error while real disconnection error.
Maybe worth to push the things in `http_send_response()` instead, but be careful with the return status handling

Reply:
Sure thing. Was wondering about this just after PR had been opened. Will update as well.